astropy/nddata/nddata_base.py

```diff
@@ -89,4 +89,12 @@ def uncertainty(self, value):
 
                 raise TypeError('Uncertainty must have attribute '
                                 'uncertainty_type whose type is string.')
+            #Try setting the parent for the uncertainty. This is currently an
+            #enforced requirement for `StdDevUncertainty` for error propagation.
+            try:
+                value.parent_nddata = self
+            except:
+                raise AttributeError('Uncertainty must have an attribute '
+                                'parent_nddata.')
         self._uncertainty = value
+        
```

Comment:
Apparently there is white space in this blank line...that is the only thing travis is complaining about right now :)